olclient/schemata/edition.schema.json

```diff
@@ -27,7 +27,7 @@
     },
     "authors": {
       "type": "array",
-      "items": { "$ref": "#definitions/author" }
+      "items": { "$ref": "shared_definitions.json#author" }
```

Comment:
*I think* these should still have the forward slash? (i.e. `shared_definitions.json#/author`). That's how they're depicted on https://spacetelescope.github.io/understanding-json-schema/structuring.html . My guess would be without the slash it's finding the first element in the json file that has key 'author' (no matter how deep), whereas with the the '/', `author` must be a child of the root of the json file. Not 100% though.